pkgs/by-name/as/astm/package.nix

```
@@ -0,0 +1,43 @@
+{ stdenv
+, gcc
+, lib
+, nlohmann_json
+, fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
Use finalAttrs pattern

---

noted